src/RecyclableMemoryStreamManager.cs

```diff
@@ -368,6 +368,12 @@ public long LargeBuffersFree
         /// set and <see cref="RecyclableMemoryStream.ToArray"/> is called, a <c>NotSupportedException</c> will be thrown.</remarks>
         public bool ThrowExceptionOnToArray { get; set; }
 
+        /// <summary>
+        /// Zero out buffers before returning them to the pool.
+        /// </summary>
+        /// <remarks>Setting this to true causes a performance hit and should only be set if one wants to avoid accidental data leaks.</remarks>
```

Comment:
This flag does more than the comment suggests. It zeros newly allocated buffers, as well as returned buffers.

Reply:
Thanks. Adjusted the comment

Reply:
Thanks. Adjusted the comment